docs/guides/prefect-deploy.md

```diff
@@ -464,6 +464,13 @@ Use the tabs below to explore these two deployment creation options.
 
     Any deployment configuration can be overridden via options available on the `prefect deploy` CLI command when creating a deployment.
 
+    !!! tip "New `prefect.yaml` file flexibility"
+        Traditionally, this file had to be in the root of your repository or project directory and named `prefect.yaml`.
+        Now this file can be located elsewhere and be named differently. To use a custom prefect.yaml file, run the new 
```

Comment:
```suggestion
        Now this file can be located elsewhere and be uniquely named. To use a custom `prefect.yaml` file, run the new 
```

Reply:
My interpretation of 'uniquely named' is that the file has to be unique among, but it's technically possible for two `prefect.yaml` files to have the same name but be in different directories. Do you think the 'uniquely named' phrasing is still the best way to describe this?

Reply:
ah, good call. I'll remove my suggestion.